react/components/Dropdown/Dropdown.css

```diff
@@ -0,0 +1,28 @@
+:global(.vtex-input__prefix) .dropdown,
+:global(.vtex-input__suffix) .dropdown {
+  height: 100%;
+}
+
+:global(.vtex-input__prefix) .dropdown {
+  margin-left: -1rem;
+}
+
+:global(.vtex-input__suffix) .dropdown {
+  margin-right: -1rem;
+}
+
+:global(.vtex-input__prefix) .container,
+:global(.vtex-input__suffix) .container {
+  height: 100%;
+  border-top: none;
+  border-bottom: none;
+  border-radius: 0;
+}
+
+:global(.vtex-input__prefix) .container {
+  border-left: none;
+}
+
+:global(.vtex-input__suffix) .container {
+  border-right: none;
+}
```

Comment:
why are all classes as global?

Reply:
because this file is transpiled to css modules, and if I use them normally here (without the `:global`) I wouldn't be able to style it correctly